lib/ex_okex/futures/public.ex

```diff
@@ -0,0 +1,9 @@
+defmodule ExOkex.Futures.Public do
+  import ExOkex.Api
```

Comment:
Thoughts on having `ExOkex.Api.Public` similar to this module's scoping so we don't send unnecessary auth headers? or having a different method other than `get`?

Reply:
👍 to `ExOkex.Api.Public`